src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -1694,6 +1733,83 @@ void CheckPendingPrefix(Prefix mask)
             Check((mask & Prefix.Constrained) == 0, VerifierError.Constrained);
         }
 
+        bool HasPendingPrefix(Prefix prefix)
+        {
+            return (_pendingPrefix & prefix) != 0;
+        }
+
+        void ClearPendingPrefix(Prefix prefix)
+        {
+            _pendingPrefix &= ~prefix;
+        }
+
+        void VerifyPendingPrefix(ILOpcode opCode)
+        {
+            if (_pendingPrefix == 0)
+                return;
+
+            if (HasPendingPrefix(Prefix.Volatile) || HasPendingPrefix(Prefix.Unaligned))
+            {
+                switch (opCode)
+                {
+                    case ILOpcode.ldind_i:
```

Comment:
I removed this method and added a check, if all prefix flags have been cleared in `EndImportingInstruction`. This will also handle the case when `unaligned` and `volatile` are used in combination.